test/dynamo/test_repros.py

```diff
@@ -1117,37 +1117,6 @@ def f(x):
         out_test.sum().backward()
         self.assertEqual(leaf.grad, leaf_test.grad)
 
-    # https://github.com/pytorch/pytorch/issues/113263
-    def test_unpack_hooks_dont_run_during_tracing(self):
```

Comment:
isn't this test still valid? we run the old path of "deferring saved tensor hooks til eager runtime" when the hooks are not GraphModules